tianshou/trainer/offpolicy.py

```diff
@@ -33,7 +33,7 @@ def offpolicy_trainer(
 ) -> Dict[str, Union[float, str]]:
     """A wrapper for off-policy trainer procedure.
 
-    The "step" in trainer means a policy network update.
+    The "step" in trainer means an environment frame.
```

Comment:
```suggestion
    The "step" in trainer means an environment step.
```

Reply:
seems in other places in doc, environment 'step' is described as frame. wouldn't it be weird that here is step?

Reply:
I agree this suggestion. Now that we have explained that the `step` refers to the environment step, all `frames` in the doc can be changed to `steps`. In addition, the annotation of `step` in offline trainer can also be deleted, because we have used the word `update` to refer to the gradient step.

Reply:
@danagi please help us check again including docstring of 3 trainers and the description of trainer in `docs/tutorials/[dqn|concepts].rst`, thanks!